src/rdkafka_request.c

```diff
@@ -5445,7 +5447,7 @@ rd_kafka_resp_err_t rd_kafka_DescribeConfigsRequest(
         }
 
         ApiVersion = rd_kafka_broker_ApiVersion_supported(
-            rkb, RD_KAFKAP_DescribeConfigs, 0, 1, NULL);
+            rkb, RD_KAFKAP_DescribeConfigs, 0, 3, NULL);
```

Comment:
I would suggest not to increase the version if it is not necessary. We would need to improve APIs as well to support that and there would be bunch of other things as well which we should be taking care of.